src/rust/ide/view/graph-editor/src/component/visualization/container/action_bar.rs

```diff
@@ -70,6 +73,144 @@ mod background {
 
 
 
+// ========================
+// === Action Bar Icons ===
+// ========================
+
+mod drag_icon {
```

Comment:
should have doc describing how the icon look like. So if you read the code you know what it is about. The same applies to the second icon

Reply:
Also, I think we should use better names for icons - `four_arrow_icon`, and also a `pin_icon`. The fact that these icons are used to drag and to reset position is based on their usage, but the definition is not really connected with the usage.